### PR TITLE
Switch genrepo to AWS

### PR DESCRIPTION
**Pull request by ambv**
Also moved the incron processor to a single Python 3 file.

Also changed the SSH ports in the integration scripts since I mistakenly set them up differently on AWS compared to the old ones on GC.

### CODE REVIEW

File: integration/macos/publish.sh [hunk 1]
```diff
@@ -14,7 +14,7 @@ set -ex
 
 cat <<EOF >"${HOME}/.ssh/config"
 Host upload-packages.edgedb.com
-    Port 2224
+    Port 2222
```
Comment (elprans) [inline integration/macos/publish.sh:17]: Curious: why the port switcheroo?

File: server/containers/genrepo/config/ssh/pub-packages-prod--root [hunk 2]
```diff
@@ -0,0 +1,3 @@
+ssh-rsa AAAAB3NzaC1yc2EAAAABIwAAAQEA1yPOwMDXwSPXTdB3u7IEnRMHGz3q+YZpwYLB1yLY6fs8ya1kZ79Ic8vi5e/7MNffeZ87zXmcf+Os/JyJNShavQxhghSiMlZeuvgelMdXgUnjkW96NR5atAI8YeJ+X1Ir7sbS4TMaBMIGqLfnQhn37I8QMk+TEgWUSxiR/Om82vnlj+c9sgTE+iyvlHLo8tNAmwoaPESAc8QzxH2WPsVmmlKpLBbTt7Wv50qrSKSdOOZOcZrHuk+pT7ZoFyxFKWDkV+i8uDtnegNtbMSsTMRgPwjoQib0FBCAMQJxabrVyj60OdZ5/FYIbWIqdM8DrQHffU3iDU5x3WvVZDuBuZu1Dw==
```
Comment (elprans) [inline server/containers/genrepo/config/ssh/pub-packages-prod--root:1]: Are these supposed to be committed?  Switching keys would be more complicated this way.

File: integration/macos/publish.sh (hunk 1 above)
Comment (ambv) [reply integration/macos/publish.sh:17]: I just set this up in the wrong order in the LB on AWS and now switching the ports up is a hassle, we'd need to recreate the LB and the service.

File: server/containers/genrepo/config/ssh/pub-packages-prod--root (hunk 2 above)
Comment (ambv) [reply server/containers/genrepo/config/ssh/pub-packages-prod--root:1]: Right, I put them here before I made storage in the SecretsManager work. I'll remove that.

**Review by elprans — APPROVED** [review]

**Pull request merged**